var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -141,9 +141,10 @@ def configure_args(self):
                        '--enable-largefile',
                        '--enable-netcdf-4']
 
-        # The flag was introduced in version 4.1.0
-        if self.spec.satisfies('@4.1:'):
+        if "+fsync" in self.spec:
             config_args.append('--enable-fsync')
+        else:
+            config_args.append('--disable-fsync')
```

Comment:
```suggestion
        config_args.extend(self.enable_or_disable('fsync'))
```

Reply:
That looks like it works for me.

Reply:
I'm fine with both ways of extending a list, so feel free to choose whatever seems more consistent.

Reply:
It does, I checked.  Much cleaner in my opinion, thanks!